docs/src/app/DocumentationMenu.js

```diff
@@ -0,0 +1,81 @@
+import React, {Component} from 'react';
+import FlatButton from 'material-ui/FlatButton';
+import Popover, {PopoverAnimationVertical} from 'material-ui/Popover';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+import {Link} from 'react-router';
+import {white} from 'material-ui/styles/colors';
+
+class DocumentationMenu extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+    };
+  }
+
+  handleTouchTap = (event) => {
+    // This prevents ghost click.
+    event.preventDefault();
+    this.setState({
+      open: true,
+      anchorEl: event.currentTarget,
+    });
+  };
+
+  handleRequestClose = () => {
+    this.setState({
+      open: false,
+    });
+  };
+
+  render() {
+    return (
+      <span>
+        <FlatButton
+          style={{color: white, position: 'relative', verticalAlign: 'middle'}}
+          onTouchTap={this.handleTouchTap}
+          label="Documentation"
+          labelPosition="before"
+          icon={<i className="material-icons">arrow_drop_down</i>}
+        />
+        <Popover
+          open={this.state.open}
+          anchorEl={this.state.anchorEl}
+          anchorOrigin={{horizontal: 'left', vertical: 'bottom'}}
+          targetOrigin={{horizontal: 'left', vertical: 'top'}}
+          onRequestClose={this.handleRequestClose}
+          animation={PopoverAnimationVertical}
+        >
+          <Menu>
+            <MenuItem
+              primaryText="Devices"
+              primary={false}
```

Comment:
When opening the Documentation menu I get a warning in the console that I hypothesize is caused by this line:

```
Warning: Unknown prop `primary` on <a> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in a
    in Link (created by EnhancedButton)
    in EnhancedButton (created by ListItem)
    in div (created by ListItem)
    in ListItem (created by MenuItem)
    in MenuItem (created by Menu)
    in div (created by List)
    in List (created by Menu)
    in div (created by Menu)
    in ClickAwayListener (created by Menu)
    in Menu
    in div (created by Paper)
    in Paper (created by PopoverAnimationVertical)
    in PopoverAnimationVertical
```